keps/NNNN-kep-template/README.md

```diff
@@ -128,7 +128,9 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 - [ ] (R) KEP approvers have approved the KEP status as `implementable`
 - [ ] (R) Design details are appropriately documented
 - [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+  - [ ] e2e Tests for all new Endpoints
```

Comment:
Can we be more explicit about what we mean by `Endpoints` and what specifically is required here?

Reply:
I expanded this out a bit, and also updated the underlying metadata to be explicit about API endpoints that are part of the KEP:

https://github.com/kubernetes/enhancements/pull/2146/files#diff-56771f8d4989d3c6f35d326519113795505560c5e277b61b667c96ab09e4067dR35-R45

Reply:
https://github.com/kubernetes/enhancements/pull/2146/files#r535576335

```yaml
# The openAPI operations to expose in api/open-apispec/swagger.json
 apis:
   - stage: alpha
     operations:
       - useThisFeatureAlphaV1
   - stage: beta
     operations:
       - useThisFeatureBetaV1
   - stage: stable
     operations:
       - useThisFeatureV1
```

Reply:
I am concerned that listing endpoints will be relatively burdensome for KEP authors, without providing a huge value to anyone outside of the conformance team.

Reply:
Removed https://github.com/ii/keps/commit/20b93a540e6f767d81d845ca053effb912ab267e